### PR TITLE
Added ipfs weekly 1 post

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/ipfs/weekly/issues/9.

### CODE REVIEW

File: src/5-ipfs-weekly-1/index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+date: 2016-01-11
+id: 5-ipfs-weekly-1
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "3-ipfs-weekly-1", link: "./" }
```
Comment (jbenet) [inline src/5-ipfs-weekly-1/index.md]: this number should match the one in the `id:` also check that 5 is the right number (im not sure, there one post there that's a draft and not published-- can bump its number up for this one, or whatever)

**Comment by jbenet** [comment]
one comment otherwise LGTM

**Comment by RichardLitt** [comment]
@jbenet Changed to 4, as the most recent published on the website is 3.

**Comment by RichardLitt** [comment]
We can change 4-registry-mirror to 5-registry-mirror when it is published.

**Comment by jbenet** [comment]
sgtm

**Pull request merged**